### DISCUSSION TITLE
Share feedback flow is painful

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49597] [Mobile] [answered]
GitHub Mobile beta

1. You have to get to the home 🏠 screen. This can take a whole bunch of taps on its own, and not always in the same place, but eventually this was a tap in the lower left corner
    ![Screenshot_20230309-015419.png](https://user-images.githubusercontent.com/2119212/223944134-fd1714ab-2ddb-4034-b975-d53f92cd3439.png)
2. Now you have to guess that you should go to your profile 👤. Why? Dunno. It's a tap in the lower right corner
    ![Screenshot_20230309-015436.png](https://user-images.githubusercontent.com/2119212/223944280-ffb2928c-90bb-4ce8-81a2-1040372e1079.png)
3. Now you have to guess that you should tap the cog ⚙️. This is the upper right corner
    ![Screenshot_20230309-015446.png](https://user-images.githubusercontent.com/2119212/223944302-c8d1c10d-a2e5-47ac-8122-d5b3b0f60c43.png)
4. Now you're looking at Mobile issues, so you guess that you can tap the ➕. This is also in the upper right corner
    ![Screenshot_20230309-015456.png](https://user-images.githubusercontent.com/2119212/223944319-bc135d87-86b8-4b57-b0e0-e46bd669fdd4.png)
5. While you were looking at the mobile discussion a moment ago, that category context is now lost 
    ![Screenshot_20230309-015505.png](https://user-images.githubusercontent.com/2119212/223944329-f5a063b9-3c04-4c62-9282-73d70a3a60c5.png)
6. So you get to scroll down to Mobile (down and left again)

This is not faster than putting community/community on the home screen, which I did ages ago.

I do not expect to use this flow ever again. I'm using it because it was advertised in a blog.

**Comment by aconsuegra** [comment]
Sorry for the late reply @jsoref. I agree that this is a painful process involving many steps, do you have any suggestion about how we could make it better?

**Reply by jsoref** [reply]
The ⋮ could always have "Send feedback" and it could go directly to the "new discussion in community/community in the category: mobile".

For any compose screen where an ✈️ appears at the top right, instead of anchoring to ⋮ there, you could put a ⋮ at the right edge of the compose toolbar.

![Screenshot_20230322-094804.png](https://user-images.githubusercontent.com/2119212/226924528-75e46dca-161d-45dc-b460-c771868cf530.png)

**Reply by jsoref** [reply]
Alternatively, just letting users tap and hold whatever item is at the top right and get a menu with send feedback would probably be better.

Otherwise you'd have to figure out how to address this kind of screen:
![Screenshot_20230322-095010.png](https://user-images.githubusercontent.com/2119212/226925143-c4da88de-f8e8-4846-a673-dadbd63a42dd.png)

**Reply by eli-schwartz** [reply]
The biggest problem for me is that it's almost entirely impossible to get to the home screen. I usually use the app by following links in other apps, and the only page that has a "home" button is, weirdly, the home screen. I may have to tap the back button *literally* hundreds of times in order to get to the home screen. This doesn't *just* affect feedback, it makes it actually impossible to use the app altogether except for opening links in another app... But it definitely makes sending feedback hard too.

Normal, user-friendly apps have a global menu of some sort, whether it's a pinned taskbar or a menu that slides out from the side or a hamburger icon in the corner or something.

**Reply by aconsuegra** [reply]
Makes sense! I think one solution would be to follow the same navigation pattern as in the iOS app, where the bottom navigation bar is always visible to the user.

**Reply by jsoref** [reply]
Whether you go with the button bar at the bottom or a sidebar, having something that's always available would definitely address a number of my complaints. This app has a really high nesting level and being forced to use back repeatedly is very painful. It doesn't help that the back button is often an "up" button and doesn't actually match the users stack. (I need to report that too, but....)

Offhand, I'd favor the hamburger (≡) because you have the entire home screen worth of things I'm likely to want to jump to and collapsing the home screen to a sidebar makes perfect sense to me. But either would be an improvement over the current state.

**Reply by jsoref** [reply]
@aconsuegra any progress on opening a ticket to do this ([make] the bottom navigation bar is always visible to the user)?

**Reply by aconsuegra** [reply, marked as answer]
We are right now exploring what it could take to bring a persistent bottom navigation bar to the Android app. Hopefully we can find an easy way to build it, and ship it soon.
